### ISSUE TITLE
Data as string vs object

### ISSUE THREAD

**Issue by tilghman** [SaeX/calendar-yearview-blocks#3]
What was the intent behind making the data parameter a string, rather than an object?

This originally tripped me up while I was implementing the module in a project.  It seems like it could be an object, directly, and it would behave a lot more like other modules.  It would also simplify the example code, as we could space the object over multiple lines, and it would wrap in the README.

**Comment by opbod** [comment]
Agreed on the fact that `Object` would provide more flexibility. I guess originally it was a string since it was easier at that time and later things were built upon that.
If you want to make the change (including the example/README), feel free!
